processing/src/main/java/org/apache/carbondata/processing/loading/sort/unsafe/merger/UnsafeIntermediateMerger.java

```diff
@@ -88,6 +88,10 @@ public UnsafeIntermediateMerger(SortParameters parameters) {
           CarbonLoadOptionConstants.CARBON_LOAD_SORT_MEMORY_SPILL_PERCENTAGE,
           CarbonLoadOptionConstants.CARBON_LOAD_SORT_MEMORY_SPILL_PERCENTAGE_DEFAULT);
       spillPercentage = Integer.valueOf(spillPercentageStr);
+      if (spillPercentage > 100 || spillPercentage < 0) {
```

Comment:
One suggestion: add a function, called 'getSortMemorySpillPercentage', in 'CarbonProperties' to get the value of 'CARBON_LOAD_SORT_MEMORY_SPILL_PERCENTAGE', and validate the value in this function. We don't need to validate this value again if other place need to use this config, just need to call 'getSortMemorySpillPercentage'.

Reply:
property validation can be validated in `CarbonProperties.validateAndLoadDefaultProperties`, please add it there. And I agree with @zzcclp , you can add a function in `CarbonProperties` to get the value of this property, just like `getInsertIntoDatasetStorageLevel`